src/models/comment.js

```diff
@@ -0,0 +1,27 @@
+export default (sequelize, DataTypes) => {
+  const Comment = sequelize.define('Comment', {
+    tripId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    comment: {
+      type: DataTypes.TEXT,
+      allowNull: false,
+    },
+    isVisible: {
+      type: DataTypes.BOOLEAN,
+      allowNull: false,
+      defaultValue: true,
+    }
+  }, {});
+  Comment.associate = (models) => {
+    Comment.belongsTo(models.Trip, {
+      foreignKey: 'tripId', as: 'trips', timestamps: false
+    });
```

Comment:
Comment should also be associated with a user, as a user would not have to input userId. Please, check this

Reply:
done